src/palace/manager/search/service.py

```diff
@@ -110,10 +110,6 @@ def read_pointer_set_empty(self) -> None:
     def index_create(self, revision: SearchSchemaRevision) -> None:
         """Atomically create an index for the given base name and revision."""
 
-    @abstractmethod
-    def indexes_created(self) -> list[str]:
-        """A log of all the indexes that have been created by this client service."""
-
```

Comment:
This method was only ever used in tests to delete created indexes. This is now handled by the search fixture, so this method can come out.